plugins/xivo-aastra/5.0.0/pkgs/pkgs.db

```diff
@@ -0,0 +1,106 @@
+[pkg_6863i-fw]
+description: Firmware for Aastra 6863i
+description_fr: Micrologiciel pour Aastra 6863i
+version: 5.0.0.1018
+files: 6863i-fw
+install: 6863i-fw
+
+[pkg_6865i-fw]
+description: Firmware for Aastra 6865i
+description_fr: Micrologiciel pour Aastra 6865i
+version: 5.0.0.1018
+files: 6865i-fw
+install: 6865i-fw
+
+[pkg_6867i-fw]
+description: Firmware for Aastra 6867i
+description_fr: Micrologiciel pour Aastra 6867i
+version: 5.0.0.1018
+files: 6867i-fw
+install: 6867i-fw
+
+[pkg_6869i-fw]
+description: Firmware for Aastra 6869i
+description_fr: Micrologiciel pour Aastra 6869i
+version: 5.0.0.1018
+files: 6869i-fw
+install: 6869i-fw
+
+[pkg_6873i-fw]
+description: Firmware for Aastra 6873i
+description_fr: Micrologiciel pour Aastra 6873i
+version: 5.0.0.1018
+files: 6873i-fw
+install: 6873i-fw
+
+[pkg_lang]
+description: Language files
+description_fr: Fichiers de langue
+version: 5.0.0.1018
+files: lang
+install: aastra-lang
+
+
+[install_aastra-fw]
+a-b: unzip $FILE1
+
+[install_6863i-fw]
+a-b: unzip $FILE1
+b-c: include *.st
+
+[install_6865i-fw]
+a-b: unzip $FILE1
+b-c: include *.st
+
+[install_6867i-fw]
+a-b: unzip $FILE1
+b-c: include *.st
+
+[install_6869i-fw]
+a-b: unzip $FILE1
+b-c: include *.st
+
+[install_6873i-fw]
+a-b: unzip $FILE1
+b-c: include *.st
+
+[install_aastra-lang]
+a-b: unzip $FILE1
+b-c: cp * i18n/
+
+
+[file_6863i-fw]
+filename: 58015370REV035001018SP16863i.zip
+url: https://www.mitel.com/-/media/58015370rev035001018sp16863i.zip
```

Comment:
404 fir the 6863i fw

Reply:
Mitel moved the files in a subdirectory during the short time it took for the review! It's fixed now.